employee_service/src/employee_repository.py

```diff
@@ -42,6 +46,8 @@ def get_employee(empid):
         except Exception as e:
             raise Exception('Error: ', e)
 
+
+
     @staticmethod
     def update_emp_details(empid, name, age, gender):
         try:
```

Comment:
the multiple if statements can be refactored based on the dictionary object passed via the API body